digi604/django-smart-selects
# Fix bogue when adding a new formset with horizontal=True

This fixes a bug when adding new formset (that use horizontal=True) using javascript (when you click the link add another...) after the page is loaded.

To verify the bug:
1. Install this branch locally and remove the modifications I have made in the file `smart_selects/static/smart-selects/admin/js/bindfields.js`
2. Go to http://127.0.0.1:8000/admin/test_app/person/ and add at least two persons 
3. Go to http://127.0.0.1:8000/admin/test_app/talent/ and add at least two talents and select differents persons that have the talent you are adding.
4. Go to http://127.0.0.1:8000/admin/test_app/group/add/ for adding a group, here try to add more than one person with talent in the membership, the current loaded inline should work as expected, but when you add a new inline using the link `Add another Membership` the new ones should not work as expected.

The steps to verify the fix are:
1. Apply the changes I have made in the `smart_selects/static/smart-selects/admin/js/bindfields.js`

Ref: issue #210
